packages/account/src/test-utils/setup-test-provider-and-wallets.ts

```diff
@@ -73,7 +73,7 @@ export async function setupTestProviderAndWallets({
       defaultSnapshotConfigs,
       walletsConfig.apply(nodeOptions?.snapshotConfig)
     ),
-    port: '0',
+    port: nodeOptions.port || '0',
```

Comment:
I intentionally made this to be always `0` so that people don't try assigning the same port in multiple tests which will cause errors. In tests, users shouldn't care on which port the node they're targeting runs. I suppose you made this change only for the purposes of [`fuel-core-schema.test.ts`](https://github.com/FuelLabs/fuels-ts/blob/9f41a255c33f7c44977aa3a161ec36aeff36c67a/packages/account/test/fuel-core-schema.test.ts).

Instead of making a change here, I suggest you change the `build:schema` script to run a node with `launchNode` and get the schema from it:
```ts
// get-fuel-core-schema.mts
const { url, cleanup } = await launchNode();

execSync(`get-graphql-schema ${url} > src/providers/fuel-core-schema.graphql && prettier --write src/providers/fuel-core-schema.graphql`);

cleanup();
```

Then you can remove this change and remove the `beforeEach`/`afterEach` blocks from `fuel-core-schema.test.ts`.

Reply:
I asked for this.

As a user, I may want to start the node on a desired port.